package.json

```diff
@@ -65,7 +65,7 @@
     "solidity-coverage": "^0.5.11",
     "solidity-multicall": "^1.0.0",
     "solium": "^1.2.4",
-    "truffle": "^5.0.26",
+    "truffle": "^5.0.44",
```

Comment:
Why upgrade?

Reply:
To support solc 0.5.7

Reply:
Is this needed for SafeTransfer? Or should this be a separate PR?

Reply:
Yes, it is needed.